### PR TITLE
Add credits section

### PR DESCRIPTION
**Pull request by qwrtln**
en-02
en-18
en-19

### CODE REVIEW

File: sections/credits.tex [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+\addsection{Credits}{\images/experience.png}
+
+\iftoggle{printable}{\vspace{-\baselineskip}}{}
+
+\bigbreak
+
+\begin{multicols*}{2}
+
+This project shamelessly borrows best practices from the \href{https://github.com/Heegu-sama/Homm3BG}{\textbf{Rule Book Rewrite Project}}, which include LaTeX code, GitHub engineering, and translation framework.
```
Comment (tomaas-zeman) [inline sections/credits.tex:11]: > shamelessly borrows

🤣

File: sections/credits.tex [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+You should check it out if you haven't seen it yet.
+
+\textbf{Scenario templating and LaTeX writing}: Tomáš Zeman, Andrzej Wiącek, Saveliy Ivanov
```
Comment (tomaas-zeman) [inline sections/credits.tex:14]: You should put yourself to be the first :)

**Review by tomaas-zeman — APPROVED** [review]

File: sections/credits.tex (hunk 2 above)
Comment (qwrtln) [reply sections/credits.tex:14]: Nah, so far I haven't added even a single scenario 😂

Comment (tomaas-zeman) [reply sections/credits.tex:14]: But you're the "father" of the project :D

Comment (qwrtln) [reply sections/credits.tex:14]: Well, time for the godfather to shine!

**Pull request merged**